www-apps/prewikka/prewikka-3.0.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 python-r1
+
+DESCRIPTION="Graphical front-end analysis console for the Prelude Framework"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/3.0.0/${PF}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="dev-python/setuptools
+	dev-python/lesscpy
+	${PYTHON_DEPS}"
+
+RDEPEND="dev-python/cheetah[${PYTHON_USEDEP}]
+	dev-python/Babel[${PYTHON_USEDEP}]
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	dev-python/pytz[${PYTHON_USEDEP}]
+	media-fonts/font-xfree86-type1
+	dev-libs/libprelude[${PYTHON_USEDEP}]
+	dev-libs/libpreludedb[${PYTHON_USEDEP}]
+	${PYTHON_DEPS}"
```

Comment:
indeed, but it is not mandatory, I should add a useflag for this. which useflag do you advise me to use ?